Source/WebCore/html/track/TextTrack.idl

```diff
@@ -51,6 +54,7 @@ enum TextTrackKind { "subtitles", "captions", "descriptions", "chapters", "metad
 
     attribute EventHandler oncuechange;
 
+    // FIXME: Remove 'non-standard'
```

Comment:
This is actually from https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/region.html#extension-of-the-texttrack-object AFAICT?

Reply:
I am going by following: https://github.com/w3c/webvtt/pull/31 , which is to remove them from TextTrack.

Reply:
and https://github.com/w3c/webvtt/commit/970b3a2d425d26e63fe2cd94b030f8f80289af00